app/models/bag_man_request.rb

```diff
@@ -39,7 +39,7 @@ def cancel!(reason)
     unless cancelled
       transaction do
         update!(cancelled: true, cancel_reason: reason)
-        replication_transfer.cancel!(reason)
+        replication_transfer.cancel!(reason, last_error)
         send_cancel(replication_transfer)
       end
     end
```

Comment:
If the jobs are disabled we use a null queue, so that'll return success.  If the jobs are enabled and resque or redis is down, that should explode which causes the transaction to be rolled back.